bin/test_integration

```diff
@@ -48,12 +49,12 @@ rm -rf $CURRENT_DIR/output/*
 # Hint: '--privileged' is added here otherwise we will get operation
 # not permitted when attempting to do anything related to dbus
 if [[ "$DEBUG" == "true" ]]; then
-  docker-compose exec --privileged test /bin/sh -c " /dbus.sh" $@
+  docker-compose exec --privileged test /bin/sh -c " /tests_executor.sh" $@
```

Comment:
I don't think test_executor is a good name here because the script is strictly for dbus configurations . I would call it `configure_dbus.sh`

@mbenita-Cyberark @eranha

Reply:
The script runs the tests at the end.